### PR TITLE
FIX:  polling; tiny behavior adjustments

### PR DESCRIPTION
**Pull request by BurntVal**

### CODE REVIEW

**Review by justinbarry — APPROVED** [review]

File: packages/abstraxion/src/hooks/useAbstraxionAccount.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export const useAbstraxionAccount = (): AbstraxionAccountState => {
       setIsConnecting(false);
     }
 
-    if ((!isConnecting && !abstraxionAccount) || !granterAddress) {
+    if (!isConnecting && !abstraxionAccount && !granterAddress) {
```
Comment (justinbarry) [inline packages/abstraxion/src/hooks/useAbstraxionAccount.ts:53]: Why this change? @BurntVal

**Review by justinbarry — APPROVED** [review]
LGTM, left one comment

File: packages/abstraxion/src/hooks/useAbstraxionAccount.ts (hunk 1 above)
Comment (BurntVal) [reply packages/abstraxion/src/hooks/useAbstraxionAccount.ts:53]: I changed it the OR in the previous PR but went back to AND because I found that it was causing an unnecessary surplus of re-renders

**Review by BurntNerve — APPROVED** [review]
Looks good!
